helm-charts/common/data-prep/README.md

```diff
@@ -38,21 +38,21 @@ Then run the command `kubectl port-forward svc/data-prep 6007:6007` to expose th
 Open another terminal and run the following command to verify the service if working:
 
 ```console
-curl http://localhost:6007/v1/dataprep  \
+curl http://localhost:6007/v1/dataprep/ingest  \
     -X POST \
     -H "Content-Type: multipart/form-data" \
     -F "files=@./README.md"
 ```
 
 ## Values
 
-| Key                    | Type   | Default                 | Description                              |
-| ---------------------- | ------ | ----------------------- | ---------------------------------------- |
-| image.repository       | string | `"opea/dataprep-redis"` |                                          |
-| service.port           | string | `"6007"`                |                                          |
-| REDIS_URL              | string | `""`                    |                                          |
-| TEI_EMBEDDING_ENDPOINT | string | `""`                    |                                          |
-| global.monitoring      | bool   | `false`                 | See ../../monitoring.md before enabling! |
+| Key                     | Type   | Default                 | Description                              |
+| ----------------------- | ------ | ----------------------- | ---------------------------------------- |
+| image.repository        | string | `"opea/dataprep-redis"` |                                          |
+| service.port            | string | `"6007"`                |                                          |
+| DATAPREP_COMPONENT_NAME | string | `"OPEA_DATAPREP_REDIS"` | vector DB backend                        |
```

Comment:
DATAPREP_BACKEND:
* "REDIS"
* "MILVUS"

could be more user friendly.

Reply:
done